ui/helper.c

```diff
@@ -550,6 +550,59 @@ void ConvertRssiToDbm(uint16_t Rssi) {
 		}	
 }
 
+void ConvertRssiToSmeter(uint16_t Rssi)
+{
+	/* S-Meter to dbm
+	S9 + 40 dB 	-53 dBm
+	S9 + 30 dB 	-63 dBm
+	S9 + 20 dB 	-73 dBm
+	S9 + 10 dB 	-83 dBm
+	S9 			-93 dBm
+	S8 			-99 dBm
+	S7 			-105 dBm
+	S6 			-111 dBm
+	S5 			-117 dBm
+	S4 			-123 dBm
+	S3 			-129 dBm
+	S2 			-135 dBm
+	S1 			-141 dBm
+	S0 			-147 dBm
+	*/
+
+	int16_t RXdBM;
+	int16_t Svalue;
+	const int16_t RXdBMover9 = -93;
+	uint8_t len;
+
+	RXdBM = (Rssi >> 1) - 160;
```

Comment:
Beken doc says
> RSSI (dBm) = REG_67<8:0>/2 – 160, **0.5 dB/step**

Shouldn't this value be divided by 2 to obtain the dB?
The same formula without division is used into `ConvertRssiToDbm` so who is right?🧐

Reply:
interesting... I'll try to test the value with and without the _divided by 2_ with an RF generator